examples/TEM_data/README.md

```diff
@@ -0,0 +1,381 @@
+TEM data Example
+================
+This example demonstrates how a user can access and combine different
+datasets and represent them in the form he or she needs.
+
+The basic idea is to semantically document both the datasets and the
+different ways one want to represent them as.
+
+The example uses DLite as the underlying interoperability framework,
+but also introduces a set of other tools for working with semantically
+documented data.
+
+
+Content
+-------
+- [Background](#background)
+- [Data resources](#data-resources)
+- [Workflow](#workflow)
+- [Setup](#setup)
+- [Running the example](#running-the-example)
+  - [Initialisation](#initialisation)
+  - [Accessing the knowledge base](#accessing-the-knowledge-base)
+  - [Listing the data resources in the knowledge base](#listing-the-data-resources-in-the-knowledge-base)
+  - [Accessing data](#accessing-data)
+- [Acknowledgments](#acknowledgments)
+
+
+Background
+----------
+This example is about studying an Al-Mg-Si aluminium alloy using
+transition electron microscopy (TEM).  The aim of such studies is to
+understand how the alloy composition and heat treatment influences the
+microstructure and how the microstructure in turn influences the alloy
+properties.  Based such understanding, new alloys with improved or
+tailored properties can be developed.
+
+In this very simple example, we will look at a single Al-Mg-Si alloy
+at two conditions, after exposing it for two different heat treatments.
+
+The first heat treatment we denote DA (for direct aged), where the
+alloy was solution heat treated for 5 min at 530C and then quenched to
+room temperature (RT) and then immediately aged for 5 hours at 185C.
+
+![profile-DA](figs/profile-DA.svg)
+
+*Figure: Heat treatment profile for condition A.  Direct aging.*
+
+Two bright-field TEM images were acquired from a sample prepared after
+the aging (marked with A in the profile above).  The first is a
+low-magnification images showing a dense distribution of needle-shaped
+beta" precipitates.
```

Comment:
```suggestion
beta precipitates.
```

Reply:
I guess i am wrong here, it is more a question whether it is right or not.

Reply:
Yes, the precipitate is actually called beta double prime. Should preferable be written with greek beta...